### PR TITLE
Fix quakespasm (broken fork, please close)

### LINKED ISSUE #2652: [quakespasm] Unable to compile
i686:
```
quakedef.h:235:10: fatal error: SDL.h: No such file or directory
 #include "SDL.h"
          ^~~~~~~
compilation terminated.
make: *** [Makefile:196: gl_refrag.o] Error 1
quakespasm failed to build: `make V=0 -j1` exited with 2
```

### PR DESCRIPTION
**Pull request by superloach**
Fixes #2652 - Add SDL dependency and a note about `-basedir` and having needed `.wad`/`.pak` files.

Sorry about those first 2 commits, my fork's kinda weird now.

### CODE REVIEW

File: packages/quakespasm.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    Dir.chdir ("Quake") do
+      system "sed -i 's,/usr/local/games/quake,#{CREW_DEST_PREFIX}/bin/,g' Makefile"
+      system "make", "install"
+    end
+  end
+
+  def self.postinstall
+    puts "Please make sure you have the proper .pak and/or .wad files to play Quake!".lightblue
+    puts "By default, quakespasm uses the current directory, but you can also specify the -basedir option.".lightblue
```
Comment (uberhacker) [inline packages/quakespasm.rb]: Can you please explain how to get these files?

**Comment by uberhacker** [comment]
@superloach:  Thank you for the update!  It looks like there is a conflict.  Can you please resolve locally and push again?

**Comment by uberhacker** [comment]
If you are having trouble with your local branch, just start over:
```
$ cd /path/to/chromebrew/packages
$ cp quakespasm.rb ~/
$ git checkout master
$ git remote add chromebrew git@github.com:skycocker/chromebrew.git
$ git pull chromebrew master
$ git branch -D fix-quakespasm
$ git checkout -b fix-quakespasm
$ cp ~/quakespasm.rb .
$ git diff
```

**Comment by superloach** [comment]
I know this is the bad way to do things... but I've been using the GitHub web interface for all of this, so those git commands don't work.
Should I just delete and recreate my fork?

**Comment by uberhacker** [comment]
You don't want to recreate your fork everytime Chromebrew is updated.  I would suggest working locally with the chromebrew remote so you can pull the most recent changes when needed.  You have the right idea to create working branches, just make sure the working branch is off of an updated master branch first. :)

**Comment by superloach** [comment]
That's the smart way to do it, haha 😅
But for now, what can I do to fix it?

**Comment by uberhacker** [comment]
If you don't want to try the command line updates I suggested above then just create a new fork.

**Pull request closed without merging**